frontend/public/style/_overrides.scss

```diff
@@ -339,3 +339,10 @@ ul {
 .pf-v6-c-truncate--no-min-width {
   --pf-v6-c-truncate--MinWidth: 0 !important;
 }
+
```

Comment:
Nit:  we try to alphabetize rules and declarations, so putting this https://github.com/openshift/console/blob/29a7a6903ecb503c242880ec4373653fe8a8edbd/frontend/public/style/_overrides.scss#L78

Reply:
Rebased and updated the position